README.rst

```diff
@@ -84,6 +84,7 @@ optional arguments:
   --only MODULE_PATH                     only include modules that start with MODULE_PATH, multiple paths can be provided
   --externals                            create list of direct external dependencies
   --reverse                              draw arrows to (instead of from) imported modules
+  --rankdir {TB,BT}                      set the rank direction for the imported modules ('TB' = top-bottom / top-down (default), 'BT' for bottom-top / bottom-up)
```

Comment:
Could you reword this to explain the effect the option has? E.g. "sets the direction of the graph, TB (default) imports flow downwards, BT = imports bubble upwards" - i.e. make it understandable for people who do not know dot.